src/main/java/uk/ac/cam/cl/dtg/isaac/api/managers/GameManager.java

```diff
@@ -980,6 +980,52 @@ public List<GameboardItem> getNextQuestionsForFilter(final GameFilter gameFilter
     return selectionOfGameboardQuestions;
   }
 
+  /**
+   * This generates a set of five question in random.
+   *
+   * @param gameFilter - to enable search
+   * @param limit      - to provide a limit of questions
+   * @return a list of questions
+   * @throws ContentManagerException - if there is a problem accessing the content repository.
+   */
+  public List<ContentDTO> generateRandomQuestions(final GameFilter gameFilter, final int limit)
```

Comment:
What you have done is not necessarily wrong, but there is also a `QuestionDTO`, which may be what you want to return instead.